### DISCUSSION TITLE
Design Issue in Load Simulation Phase

### DISCUSSION THREAD

**Discussion by KomalAgarwal1** [artilleryio/artillery#1499] [Artillery]
Hi Team,

I have a scenario like below:

Ramp up: 100 Vusers gradually over 20 min
Steady State : Maintain steady state of 1 hour for only these 100 Vusers
Ramp down : 100 Vusers gradually over 15 min.

As per Artillery documentation we can use arrivalRate and arrivalCount  but that will have new users getting added over a period of specified duration.
Could you please help me understand how we can  maintain a steady state for 1 hour with only 100 VUsers (without new users getting added.)

**Comment by hassy** [comment]
Hi @KomalAgarwal1 👋 

What would real users (that Artillery VUs will be simulating) be doing during that one hour time window? And how do you define steady state for your tests?

Also, are you looking to emulate a scenario where users arrive over a period of 20 minutes, but don't really perform any actions until they're all ready (i.e. ~20 minutes into the test run).

If your Artillery scenario is made up of actions that run for an hour, then it's a matter of something like this:

```
- duration: 1200 # 20 minutes
  arrivalCount: 100
```

(The overlap between ALL users will be ~40m rather than one hour.)

**Reply by KomalAgarwal1** [reply]
Hi @hassy,

Thank you for your quick response.

I am trying to achieve concurrency (constant users performing different activities) once all users ramped up in the system.
Generally during ramp ups, the response time for the flows are spiked due to high load on the system.
I wanted to test once the app is stable with consistent number of users.

With the scenario you have defined, I can use that to ramp up my users.
 - duration: 1200 # 20 minutes
  arrivalCount: 100

Can you please help if there is a way to achieve the above scenario via artillery?
